cf-macro/template.yaml

```diff
@@ -21,18 +21,10 @@ Resources:
   MacroLambdaFunction:
     Type: AWS::Serverless::Function
     Properties:
-      Handler: cf-macro/index.handler
+      Handler: dist/index.handler
       Runtime: nodejs16.x
       Timeout: 60
       Architectures:
         - x86_64
       Policies:
         - AWSLambdaBasicExecutionRole
-    Metadata:
```

Comment:
Let's try and restore the SAM native way of building and minifying the app. If there is an issue with it, we can discuss and see if we can figure it out without adding manual build steps.